vm/src/obj/objbytes.rs

```diff
@@ -328,6 +329,29 @@ impl PyBytesRef {
             .ctx
             .new_bytes(self.inner.strip(chars, ByteInnerPosition::Right, vm)?))
     }
+
+    #[pymethod(name = "__mul__")]
+    fn repeat(self, n: isize, vm: &VirtualMachine) -> PyResult {
```

Comment:
I should put as many logic as possible inside a `repeat` or a `mul` method in PyByteInner.
I think it should accept a `PyIntRef` instead of isize.

Reply:
ok